python/prompt-management/temp.py

```
@@ -0,0 +1,160 @@
+            values_str = ", ".join(values)
+            insert_statements.append(f"INSERT INTO {table_name} VALUES ({values_str});")
+            
+        except Exception as e:
+            print(f"Error processing row {idx}: {e}")
+            print(f"Problematic value sample: {str(row.iloc[0])[:200]}...")
+            continue
+    
+    return insert_statements
+
+# Try different CSV reading strategies
+csv_file = "/Users/akshay/Desktop/queries.csv"
```

Pylint (3.3.7)

[convention] Constant name "csv_file" doesn't conform to UPPER_CASE naming style

(C0103)